pkg/migration/manager.go

```diff
@@ -109,9 +110,34 @@ func NewManager(
 //
 // TODO(irfansharif): We want to eventually be able to send out a Migrate
 // request spanning the entire keyspace. We'll need to make sure all stores have
-// synced once to persist any raft command applications. Implement this.
+// synced once to persist any raft command applications.
 //lint:ignore U1001 unused
-func (h *Helper) IterateRangeDescriptors(f func(...roachpb.RangeDescriptor) error) error {
+func (h *Helper) IterateRangeDescriptors(
```

Comment:
Move (and postpone) this whole commit please. Let's keep the helper as small as possible for now and add these gadgets in later when we have real migrations to work with.

Reply:
Tacked along with a real migration in #57445.